PurchasesTests/SystemInfoTests.swift

```diff
@@ -0,0 +1,73 @@
+import XCTest
+import Nimble
+
+import Purchases
+
+class SystemInfoTests: XCTestCase {
+    func testProxyURL() {
```

Comment:
I'm not testing all methods here, because some are pretty tricky and might not even be worth testing. 
an example would be the isSandbox method, since the bundle that tests use isn't even the main bundle, and it would always be sandbox in a testing environment. 
We might add the tests some day, but they didn't feel worth delaying this feature any further.